includes/Core/Consent_Mode/Consent_Mode.php

```diff
@@ -88,12 +102,31 @@ function () use ( $consent_mode_enabled ) {
 		add_filter( 'googlesitekit_inline_base_data', $this->get_method_proxy( 'inline_js_base_data' ) );
 	}
 
+	/**
+	 * Enqueues the consent mode script.
+	 *
+	 * @since n.e.x.t
+	 */
+	protected function enqueue_script() {
+		list( $filename ) = Manifest::get( 'consent-mode' );
```

Comment:
In normal usage, assets don't need to touch the manifest directly. Please see my comment on https://github.com/google/site-kit-wp/issues/8955#issuecomment-2206167344

Reply:
Thanks @aaemnnosttv, yes that's right. [I've corrected this](https://github.com/google/site-kit-wp/pull/8893/commits/16d175dd1b1cb2279a8a06e75fbe0c28093a05b6)